<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResizeDialog</class>
 <widget class="QDialog" name="ResizeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resize</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="rbSetLongest">
         <property name="text">
          <string>Set longest edge size</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Longest edge limit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbLongEdge">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>px</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="rbSetXY">
         <property name="text">
          <string>Set height, width</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Image size by pixel</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Height</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbHeight">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>px</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Width</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbWidth">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>px</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="cbKeepRatio">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Keep aspect ratio</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
         <zorder></zorder>
         <zorder></zorder>
         <zorder>cbKeepRatio</zorder>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Estimate file size &lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbFileSize">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>K</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Quality</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbQuality">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Output dir</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="rbSameDir">
          <property name="text">
           <string>Output to same dir as input files</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup_2</string>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QRadioButton" name="rbSelectDir">
            <property name="text">
             <string>Select output dir</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_2</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leOutDir"/>
          </item>
          <item>
           <widget class="QPushButton" name="pbChoose">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbApplyToAll">
       <property name="text">
        <string>Apply to all</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ResizeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ResizeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbSetLongest</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbLongEdge</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbSetXY</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbHeight</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbSetXY</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbWidth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbSetXY</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbKeepRatio</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbSetLongest</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbApplyToAll</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbSetXY</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbHeight</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbSetLongest</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbLongEdge</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbSelectDir</sender>
   <signal>toggled(bool)</signal>
   <receiver>leOutDir</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbKeepRatio</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbFileSize</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
